pkg/api/controller.go

```diff
@@ -3006,6 +3011,62 @@ func (c *Controller) GetCurrentUser(w http.ResponseWriter, r *http.Request) {
 	writeResponse(w, http.StatusOK, response)
 }
 
+func (c *Controller) sendResetPasswordEmail(email []string, token string) error {
```

Comment:
can accept just an email and not mailing list

Reply:
Fixed